packages/react/src/Label/Label.module.css

```diff
@@ -0,0 +1,168 @@
+  padding-right: var(--base-size-16);
+  height: var(--base-size-28);
+}
+
+.Label--size-large {
+  padding-left: var(--base-size-16);
+  padding-right: var(--base-size-16);
+  height: var(--base-size-32);
+}
+
+.Label--color-default {
+  --brand-Label-color: var(--brand-color-text-default);
+}
+
+.Label--color-blue {
+  --brand-Label-color: var(--brand-Label-color-blue);
+}
+
+.Label--color-blue-purple {
+  --brand-Label-color-start: var(--brand-Label-color-blue-purple-start);
+  --brand-Label-color-end: var(--brand-Label-color-blue-purple-end);
+}
+
+.Label--color-coral {
+  --brand-Label-color: var(--brand-Label-color-coral);
+}
+
+.Label--color-green {
+  --brand-Label-color: var(--brand-Label-color-green);
+}
+
+.Label--color-green-blue {
+  --brand-Label-color-start: var(--brand-Label-color-green-blue-start);
+  --brand-Label-color-end: var(--brand-Label-color-green-blue-end);
+}
+
+.Label--color-gray {
+  --brand-Label-color: var(--brand-Label-color-gray);
+}
+
+.Label--color-indigo {
+  --brand-Label-color: var(--brand-Label-color-indigo);
+}
+
+.Label--color-lemon {
+  --brand-Label-color: var(--brand-Label-color-lemon);
+}
+
+.Label--color-lime {
+  --brand-Label-color: var(--brand-Label-color-lime);
+}
+
+.Label--color-orange {
+  --brand-Label-color: var(--brand-Label-color-orange);
+}
+
+.Label--color-pink {
+  --brand-Label-color: var(--brand-Label-color-pink);
+}
+
+.Label--color-pink-blue {
+  --brand-Label-color-start: var(--brand-Label-color-pink-blue-start);
+  --brand-Label-color-end: var(--brand-Label-color-pink-blue-end);
+}
+
+.Label--color-purple {
+  --brand-Label-color: var(--brand-Label-color-purple);
+}
+
+.Label--color-purple-red {
+  --brand-Label-color-start: var(--brand-Label-color-purple-red-start);
+  --brand-Label-color-end: var(--brand-Label-color-purple-red-end);
+}
+
+.Label--color-red {
+  --brand-Label-color: var(--brand-Label-color-red);
+}
+
+.Label--color-red-orange {
+  --brand-Label-color-start: var(--brand-Label-color-red-orange-start);
+  --brand-Label-color-end: var(--brand-Label-color-red-orange-end);
+}
+
+.Label--color-teal {
+  --brand-Label-color: var(--brand-Label-color-teal);
+}
+
+.Label--color-yellow {
+  --brand-Label-color: var(--brand-Label-color-yellow);
+}
+
+.Label__text {
+  background: var(--brand-Label-background);
+  -webkit-background-clip: text;
+  background-clip: text;
+  -webkit-text-fill-color: transparent;
+  -webkit-box-decoration-break: clone;
+  box-decoration-break: clone;
+}
+
+.Label--size-small .Label__text {
+  line-height: var(--brand-text-lineHeight-100);
+}
+
+.Label--size-medium .Label__text {
+  line-height: var(--brand-text-lineHeight-200);
+}
+
+.Label--size-large .Label__text {
+  line-height: var(--brand-text-lineHeight-300);
+}
+
+.Label__icon-visual {
+  display: flex;
+  align-self: center;
+  height: var(--base-size-16);
+  width: auto;
+}
+
+.Label__leading-visual {
+  display: flex;
+  margin-right: var(--base-size-8);
+}
+
+.Label--label {
```

Comment:
Nitpick: Not sure about this class name, as it is not used as a modifier for the `Label` block.

Reply:
Good point. I'll rename it to `Label-label`